clients/cpp-console/src/common/DialogManager.cpp

```diff
@@ -247,14 +248,38 @@ void DialogManager::StartKws()
 void DialogManager::StartListening()
 {
     log_t("Now listening...");
-    if(_bargeInSupported)
+    if (_bargeInSupported)
     {
         _player->Stop();
     }
     DeviceStatusIndicators::SetStatus(DeviceStatus::Listening);
     auto future = _dialogServiceConnector->ListenOnceAsync();
 }
 
+void DialogManager::StopListening()
+{
+    log_t("Now stop listening...");
+
+    auto future = _dialogServiceConnector->DisconnectAsync();

```

Comment:
First thing you need to do here is stop playback. Is this done here?

Reply:
Updated.